src/include/duckdb/storage/serialization/nodes.json

```diff
@@ -604,61 +604,61 @@
         "type": "vector<bool>"
       },
       {"id": 117,
-        "name": "rejects_table_name",
-        "type": "string"
+        "name": "store_rejects",
+        "type": "CSVOption<bool>"
```

Comment:
We can't change the type of fields, unless the serialization of the types is identical